scripts/run_rllib_slurm.sh.template

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
-#SBATCH -p research
-#SBATCH --nodes=2
+#SBATCH -p compute
```

Comment:
Please consider adding comments to explain the directive would go a long way in helping new slurm users get started quickly.

Reply:
New users can simply take a SLURM crash course. SLURM is a universal tool that everybody using HPC should be familiar with.